modules/local/summarize_results.nf

```diff
@@ -0,0 +1,56 @@
+process SUMMARIZE_RESULTS {
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pyopenms:3.3.0--py313h9b5bd11_0' :
+        'biocontainers/pyopenms:3.3.0--py313h9b5bd11_0' }"
+
+    input:
+    tuple val(meta), path(file)
+
+    output:
+    path '*_histogram_mz.csv'              , emit: hist_mz, optional: true
+    path '*_histogram_rt.csv'              , emit: hist_rt, optional: true
+    path '*_histogram_scores.csv'          , emit: hist_scores, optional: true
+    path '*_histogram_xcorr_scores.csv'    , emit: hist_xcorr, optional: true
+    path '*_peptide_length.csv'            , emit: lengths, optional: true
+    path '*_general_stats.csv'             , emit: stats
+    path '*.tsv'                           , emit: final_tsv
+    path 'versions.yml'                    , emit: versions
+
+    script:
+        def args = task.ext.args ?: ''
+        def prefix = task.ext.prefix ?: "${meta.id}"
+        def quantify = params.quantify ? '--quantify' : ''
+
+        """
+        summarize_results.py \\
+            --input $file \\
+            --out_prefix $prefix \\
+            $quantify \\
+            $args
+
+        cat <<-END_VERSIONS > versions.yml
+        "${task.process}":
+            pyOpenMS: \$(pip show pyopenms | grep Version | cut -d ' ' -f 2)
```

Comment:
Small nitpick: Please align with this
https://github.com/nf-core/mhcquant/blob/d60fa67cc81f45f1199fa7ab3380f6204fe9dfe8/modules/local/pyopenms_ionannotator.nf#L54C1-L58C12, else we have unnecessary duplications in versions in multiqc report